dbms/src/Flash/Mpp/ReceivedMessageQueue.h

```diff
@@ -55,6 +57,65 @@ enum class ReceiverMode
     Async
 };
 
+class GRPCNotifyQueue : public NotifyFuture
+{
+public:
+    template <typename... Args>
+    explicit GRPCNotifyQueue(const LoggerPtr & log_, Args &&... args)
+        : queue(log_, std::forward<Args>(args)...)
+    {}
+
+    void registerTask(TaskPtr && task) override { queue.registerPipeReadTask(std::move(task)); }
```

Comment:
maybe rename it to `registerPipelineReadTask`?

Reply:
Because this is an override of NotifyFuture::registerTask, it cannot be renamed